pkg/apis/aerogear/v1alpha1/types.go

```diff
@@ -50,25 +50,25 @@ type KeycloakSpec struct {
 type KeycloakRealm struct {
 	ID                string                     `json:"id,omitempty"`
 	Realm             string                     `json:"realm,omitempty"`
-	Enabled           bool                       `json:"enabled,omitempty"`
-	DisplayName       string                     `json:"displayName,omitempty"`
+	Enabled           bool                       `json:"enabled"`
+	DisplayName       string                     `json:"displayName"`
 	Users             []KeycloakUser             `json:"users,omitempty"`
```

Comment:
Slices have to have the `omitempty` because they default to null, which fails the CR validation